requirements-dev.txt

```diff
@@ -4,4 +4,4 @@ flake8
 flake8-import-order
 mypy
 bandit
-pylint
+pylint
```

Comment:
This is weird. What changed here?. You used another type of newline?

Reply:
Newline at end of file